Contributing.md

```diff
@@ -0,0 +1,212 @@
+7.	Submit the pull request.
+
+One of the site administrators will process your pull request. Your pull request will surface on the OfficeDev/<repo name> site under Issues. When the pull request is accepted, the issue will be resolved.
+
+#### Create a new branch after merge
+
+After a branch is successfully merged (that is, your pull request is accepted), don't continue working in that local branch. This can lead to merge conflicts if you submit another pull request. To do another update, create a new local branch from the successfully merged upstream branch, and then delete your initial local branch.
+
+For example, if your local branch X was successfully merged into the OfficeDev/microsoft-graph-docs master branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/microsoft-graph-docs master branch. To do this, open GitBash and execute the following commands:
+
+    cd microsoft-graph-docs
+    git pull upstream master:X2
+    git push origin X2
+
+You now have local copies (in a new local branch) of the work that you submitted in branch X. The X2 branch also contains all the work other writers have merged, so if your work depends on others' work (for example, shared images), it is available in the new branch. You can verify that your previous work (and others' work) is in the branch by checking out the new branch...
+
+    git checkout X2
+
+...and verifying the content. (The `checkout` command updates the files in `C:\Users\<yourusername>\microsoft-graph-docs` to the current state of the X2 branch.) Once you check out the new branch, you can make updates to the content and commit them as usual. However, to avoid working in the merged branch (X) by mistake, it's best to delete it (see the following **Delete a branch** section).
+
+#### Delete a branch
+
+Once your changes are successfully merged into the main repository, delete the branch you used because you no longer need it.  Any additional work should be done in a new branch.  
+
+#### To delete a branch
+
+1.	In the Git Bash command prompt, type `git checkout master`. This ensures that you aren't in the branch to be deleted (which isn't allowed).
+2.	Next, at the command prompt, type `git branch -d <branch name>`. This deletes the branch on your computer only if it has been successfully merged to the upstream repository. (You can override this behavior with the `–D` flag, but first be sure you want to do this.)
+3.	Finally, type `git push origin :<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your github fork.  
+
+Congratulations, you have successfully contributed to the project!
+
+## How to use Markdown to format your topic
+
+### Standard Markdown
+
+All of the articles in this repository use Markdown. A complete introduction (and listing of all the syntax) can be found at [Markdown Home](http://daringfireball.net/projects/markdown/ 
+).
```

Comment:
Aren't we also using some MSFT specific Markdown? Like `> [!NOTE]`

Reply:
@AlexJerabek thanks for the feedback!  

I intentionally omitted any mention of the auto-generation story, as it's likely just to confuse folks. From a customer-experience standpoint, it's preferable to have folks just submit an Issue in this repo if/when there's something that needs to be updated in the auto-generated docs, and then the burden is on us to figure out where the update needs to be made (metadata files or DefinitelyTyped repo) and ensure that the docs are updated accordingly. 

Re your comments about the **Contributing.md** file -- good catch on those things. This is the  standard `Contributing.md` file that we've been using across most, if not all, of our `OfficeDev` repos for some time now. (I actually copied it over directly from `OfficeDev/office-js-docs-pr`.)  I'll update it here per your feedback, and in our team's other repos as well.  @lindalu-MSFT @davidchesnut should be able to coordinate updating this file in any other `OfficeDev` repos where it exists.

@lindalu-MSFT -- thanks for the link! I agree that page contains good info about OPS-flavored markdown, but unfortunately that page is only available internally within Microsoft so we can't include it in any guidance that we share with customers on GitHub (e.g., in **Contributing.md**).

Reply:
Re the comment about "Standard" markdown (vs OPS-flavored markdown) -- OPS-flavored markdown is 98% the same as the markdown that's described in the page that's linked to (daringfireball.net) and AFAIK we don't have any public-facing docs about OPS-flavored markdown, so that link is the best we can do, for now. I'll follow-up with @davidchesnut about this as well.